README.md

```diff
@@ -22,22 +22,28 @@ Links:
 
 # Quick Start
 ## Installation
+
 ```
-pip install fairseq2==0.1
 pip install .
 ```
 
+A temporary extra requirement for fairseq2 is [libsndfile](https://github.com/libsndfile/libsndfile). From [Conda](https://docs.conda.io/en/latest/) environment it can be installed via:
+```
+conda install -y -c conda-forge libsndfile
+```
+At this point fairseq2 has a confirmed support only for Linux and macOS.
```

Comment:
Shall we make it explicit that macOS support ✅ but pre-built package ❌, as Can mentioned in https://github.com/facebookresearch/seamless_communication/issues/19 
> right now we don't have pre-built packages available for macOS. For the time-being I suggest using a Linux box. We plan to have macOS wheels available pretty soon.